projects/azure/base-images/test-base.yaml

```diff
@@ -28,72 +28,63 @@ items:
       dockerfile: |
         FROM docker.io/centos:7
 
-        ENV HOME=/tmp/home \
-            USER_UID=1001 \
-            GOROOT=/usr/local/go \
-            GOPATH=/go \
-            GOCACHE=/go/.cache
-        ENV PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+        RUN yum -y update
 
-        # install golang-1.11
-        RUN mkdir -p /go /tmp/home && chmod -R 777 /go /tmp/home /src && \
-            curl https://storage.googleapis.com/golang/go1.11.2.linux-amd64.tar.gz | tar -C /usr/local -xzf -
+        RUN yum -y install epel-release
 
-        # install glide
-        RUN curl -L https://github.com/Masterminds/glide/releases/download/v0.13.1/glide-v0.13.1-linux-amd64.tar.gz | tar -C /tmp/ -xzf -
-        RUN mv /tmp/linux-amd64/glide /usr/bin
+        RUN rpm --import https://packages.microsoft.com/keys/microsoft.asc
+        RUN sh -c 'echo -e "[azure-cli]\nname=Azure CLI\nbaseurl=https://packages.microsoft.com/yumrepos/azure-cli\nenabled=1\ngpgcheck=1\ngpgkey=https://packages.microsoft.com/keys/microsoft.asc" >/etc/yum.repos.d/azure-cli.repo'
 
-        # install tooling
-        RUN yum update -y && \
-            yum install epel-release -y && \
-            yum install -y --setopt=skip_missing_names_on_install=False \
+        RUN yum -y install --setopt=skip_missing_names_on_install=False \
+            azure-cli \
             gcc \
             git \
             jq \
             lsof \
             make \
+            net-tools \
+            strace \
             tar \
             wget \
-            which \
-            && \
-            # enable git clone with random user (glide dependency)
-            git config --system core.logallrefupdates false
-
-        # install azure cli
-        RUN rpm --import https://packages.microsoft.com/keys/microsoft.asc && \
-            sh -c 'echo -e "[azure-cli]\nname=Azure CLI\nbaseurl=https://packages.microsoft.com/yumrepos/azure-cli\nenabled=1\ngpgcheck=1\ngpgkey=https://packages.microsoft.com/keys/microsoft.asc" > /etc/yum.repos.d/azure-cli.repo' && \
-            yum install azure-cli -y
+            which
 
         RUN yum clean all
 
+        # install golang-1.11
+        RUN curl -s https://storage.googleapis.com/golang/go1.11.6.linux-amd64.tar.gz | tar -C /usr/local -xz
+        ENV GOPATH=/go
+        ENV GOCACHE=/go/.cache
+        ENV PATH=$PATH:/usr/local/go/bin:/go/bin
+
+        RUN go get golang.org/x/tools/cmd/goyacc
+        RUN go get github.com/golang/mock/mockgen
+        RUN go get github.com/Masterminds/glide
+
+        # pin go-bindata to 41975c0, master got broken on 1/4/2019
+        RUN go get -d github.com/go-bindata/go-bindata/go-bindata
+        RUN cd /go/src/github.com/go-bindata/go-bindata && git checkout -q 41975c0
+        RUN go install github.com/go-bindata/go-bindata/go-bindata
+
+        # pin goimports to 04b5d21e, see discussion https://github.com/openshift/openshift-azure/pull/862
```

Comment:
I tempted to start migrating our code to use the new goimport format. Considering half of our code is already done